test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,47 @@
+require "test_helper"
+
+describe RecipesController do
+  it "can get the search path" do
+    VCR.use_cassette('chicken') do
+      get root_path, params: {query: "chicken meatballs"}
+
+      must_redirect_to results_path("chicken meatballs")
+    end
+  end
+
+  it "can get the recipe show view" do
+    VCR.use_cassette('Slider') do
+      slider_uri = "#recipe_3765ec2ac5aa6523a30b59203fe64883"
+      get show_path(slider_uri)
+
+      must_respond_with :success
+    end
+  end
+
+  it "handles an invalid request" do
+    VCR.use_cassette('Slider') do
+      slider_uri = "#recipe_slider"
+      proc { get show_path(slider_uri)}.must_raise
```

Comment:
I like that you've identified this as a test case - good work! However, I'm not sure I agree with your engineering decision about how the app ought to behave in this case. In general, controllers shouldn't ever expose exceptions to the user, but should send back an appropriate status code and let the user know what happened via flash.